individual-files/ProcessCommmand.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Diagnostics;
+  
+class ProcessCommand {
+    public ProcessCommand(){
+        Console.WriteLine("Enter OS command: ");
+        string userCommand = Console.ReadLine();
+        Console.WriteLine(userCommand);
+        if (userCommand.Length != 0)
+            try
+            {
+                Process.Start(userCommand);
```

Comment:
Security Defect (S): This another instance of allowing a user defined string to be part of a command to be executed.

Rank 1 - Must fix now